code/game/objects/items/tools/synth_bracer/synth_bracer.dm

```diff
@@ -0,0 +1,444 @@
+		user.update_inv_gloves()
+	if(internal_transmitter.attached_to == attacker)
+		internal_transmitter.attackby(attacker, user)
+		return
+
+	if(istype(attacker, /obj/item/device/simi_chip))
+		var/obj/item/device/simi_chip/new_chip = attacker
+		var/chips_num = 0
+		for(var/obj/item/device/simi_chip/chip in ability_chips)
+			if(chip.chip_action == new_chip.chip_action)
+				to_chat(user, SPAN_WARNING("[src] already has this circuit chip!"))
+				return
+			if(!chip.secret)
+				chips_num++
+		if(chips_num >= ability_chips_max)
+			to_chat(user, SPAN_WARNING("[src] can't hold another circuit chip!"))
+			return
+		if(user.drop_held_item())
+			new_chip.forceMove(src)
+			ability_chips += new_chip
+			to_chat(user, SPAN_NOTICE("You slot [new_chip] into [src]!"))
+			playsound(src, 'sound/machines/terminal_processing.ogg', 35, TRUE)
+			if(user.gloves && (user.gloves == src))
+				update_actions(SIMI_ACTIONS_RELOAD, user)
+			else
+				update_actions(SIMI_ACTIONS_RELOAD)
+		return
+
+	if(HAS_TRAIT(attacker, TRAIT_TOOL_SCREWDRIVER))
+		//Remove ability chip - give option or remove all?
+		var/turf/T = get_turf(user)
+		if(!T)
+			to_chat(user, "You cannot do that here.")
+			return
+
+		var/removed_chips = FALSE
+		for (var/obj/item/device/simi_chip/chip in ability_chips)
+			if(chip.secret)
+				continue
+			chip.forceMove(T)
+			ability_chips -= chip
+			removed_chips = TRUE
+		if(removed_chips)
+			if(user.gloves && (user.gloves == src))
+				update_actions(SIMI_ACTIONS_RELOAD, user)
+			else
+				update_actions(SIMI_ACTIONS_RELOAD)
+			to_chat(user, SPAN_NOTICE("You pop out the circuit chips from [src]!"))
+		else
+			to_chat(user, SPAN_NOTICE("There are no removable circuit chips in [src]!"))
+		return
+	return ..()
+
+/obj/item/clothing/gloves/synth/attack_hand(mob/living/carbon/human/user)
+	if(istype(user) && user.gloves == src)
+		internal_transmitter.attack_hand(user)
+		return
+	return ..()
+
+/obj/item/clothing/gloves/synth/forceMove(atom/dest)
+	. = ..()
+	if(isturf(dest))
+		internal_transmitter.set_tether_holder(src)
+	else
+		internal_transmitter.set_tether_holder(loc)
+//#############################
+//###### ICON HANDLING ########
+//#############################
+/obj/item/clothing/gloves/synth/update_icon()
+	var/mob/living/carbon/human/wearer = loc
+	if(!istype(wearer) || wearer.gloves != src)
+		overlays.Cut()
+		icon_state = "bracer_[bracer_color]"
+		return
+
+	icon_state = "bracer_[bracer_color]_blank"
+
+	update_overlays()
+
+/obj/item/clothing/gloves/synth/proc/get_bracer_status()
+	if(battery_charge <= 0)
+		internal_transmitter.enabled = FALSE
+		return "status_nobattery"
+	if(battery_charge <= battery_charge_max * 0.1)
+		return "status_lowbattery"
+	var/mob/living/carbon/human/wearer = loc
+	if(!issynth(wearer) && !human_adapted)
+		internal_transmitter.enabled = FALSE
+		return "status_unauthorized"
+	internal_transmitter.enabled = TRUE
+	return "status_idle"
```

Comment:
all these strings should be defines, same as above at start of PR which was perfect